### ISSUE TITLE
Tree layouts support

### ISSUE THREAD

**Issue by radumg** [ksobon/Mandrill#18]
Hey Konrad, have you thought of supporting d3.layout.tree ?

Scenarios where this would be useful could include : 

- visualisation of serialised information
- nested list exploration/display
- visualisation of workflows/instructions

What do you think ?

**Comment by ksobon** [comment]
I like that! If we can come up with a sample data that i can test this on then i can go for it.

**Comment by radumg** [comment]
hey @ksobon , here's an example built by Modelab, pretty much does what I was thinking of in terms of `nested list exploration`.
It also uses D3.js.

**Comment by ksobon** [comment]
Yeah, I got sidetracked with something else, but I am thinking about updating Mandrill at some point. Thanks!

**Comment by radumg** [comment]
no worry/hurry, just wanted to leave you a link/record for when you do get around to it.
